pa11ycrawler/settings.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+"""
+Scrapy settings for a11y project
+
+For simplicity, this file contains only settings considered important or
+commonly used. You can find more settings consulting the documentation:
+
+    http://doc.scrapy.org/en/latest/topics/settings.html
+    http://scrapy.readthedocs.org/en/latest/topics/downloader-middleware.html
+    http://scrapy.readthedocs.org/en/latest/topics/spider-middleware.html
+"""
+
+# Custom crawler settings --------------------------------------
+# Note: PA11YCRAWLER_START_URLS must be set
+
+PA11YCRAWLER_START_URLS = []
+PA11YCRAWLER_ALLOWED_DOMAINS = []
+PA11YCRAWLER_DENY_URL_MATCHER = []
+PA11YCRAWLER_REPORTS_DIR = 'pa11ycrawler_reports'
+PA11YCRAWLER_SAVE_SOURCE = False
+PA11YCRAWLER_KEEP_EXISTING_REPORTS = False
+PA11YCRAWLER_CUSTOM_REPORTER_CLASS = 'pa11ycrawler.reporter.Pa11yGenericReporter'
+
+# The following items are pa11y settings -----------------------
+# See: https://github.com/springernature/pa11y#command-line-interface
+
+# -s, --standard <name>
+# the accessibility standard to use:
+# Section508, WCAG2A, WCAG2AA (default), WCAG2AAA
+PA11Y_STANDARD = 'WCAG2AA'
+
+# -r, --reporter <reporter>
+# the reporter to use: cli (default), csv, html, json, 1.0-json
+# The default for the crawler requires
+# https://github.com/springernature/pa11y-reporter-1.0-json
+PA11Y_REPORTER = 'html'
+
+# -l, --level <level>
+# the level of message to fail on (exit with code 2): error, warning, notice
```

Comment:
These will just use whatever Pa11y's default value for the option is if the value is empty in this file and it's not set with the command line option.  I'll add a comment to this file to clarify.